src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -317,9 +322,17 @@ private function renderRowSeparator()
             return;
         }
 
-        $markup = $this->style->getCrossingChar();
+        list($leftChar, $midChar, $rightChar) = self::SEPARATOR_MIDDLE === $type
+            ? array($this->style->getCrossingMidLeftChar(), $this->style->getCrossingChar(), $this->style->getCrossingMidRightChar())
+            : (self::SEPARATOR_TOP === $type
+                ? array($this->style->getCrossingTopLeftChar(), $this->style->getCrossingTopMidChar(), $this->style->getCrossingTopRightChar())
+                : array($this->style->getCrossingBottomLeftChar(), $this->style->getCrossingBottomMidChar(), $this->style->getCrossingBottomRightChar()))
```

Comment:
this nesting of ternary operators are hard to read IMO. I suggest using a `if/elseif/else` instead (and btw, this will also avoid the need to allocate arrays holding values just to unpack it immediately)

Reply:
Is it better now?